File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
    
    const result = new Array(52).fill(0);
    
    my_string.split("").forEach((char)=> {
        const index = alphabet.indexOf(char);
        
        if (index !== -1){
            result[index] += 1;
        }
    });
    
    return result
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
# def solution(my_string):
#     alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
#     answer = [0] * 52
    
#     for n in my_string:
#         a = alphabet.index(n)
#         answer[a] += 1
    
#     return answer

def solution(my_string):
    alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    return [my_string.count(c) for c in alphabet]